pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -2874,6 +2874,25 @@ public double getLoadBalancerBandwidthOutResourceWeight() {
     )
     private boolean loadBalancerMultiPhaseBundleUnload = true;
 
+    @FieldContext(
+            dynamic = false,
+            category = CATEGORY_LOAD_BALANCER,
+            doc = "Name of ServiceUnitStateTableView implementation class to use"
+    )
+    private String loadManagerServiceUnitStateTableViewClassName =
+            "org.apache.pulsar.broker.loadbalance.extensions.channel.ServiceUnitStateTableViewImpl";
+
+    @FieldContext(
+            dynamic = true,
+            category = CATEGORY_LOAD_BALANCER,
+            doc = "Specify ServiceUnitTableViewSyncer to sync service unit(bundle) states between metadata store and "
+                    + "system topic table views during migration from one to the other. One could enable this"
+                    + " syncer before migration and disable it after the migration finishes. "
+                    + "It accepts `MetadataStoreToSystemTopicSyncer` or `SystemTopicToMetadataStoreSyncer` to "
+                    + "enable it. Null value disables it."
+    )
+    private ServiceUnitTableViewSyncerType loadBalancerServiceUnitTableViewSyncer = null;
```

Comment:
This config is inconsistent with the PIP-378, see https://github.com/apache/pulsar/blob/master/pip/pip-378.md#configuration

> Add a loadBalancerServiceUnitTableViewSyncerEnabled configuration to to enable ServiceUnitTableViewSyncer to sync metadata store and system topic ServiceUnitStateTableView during migration.

It makes sense because we have two syncer implementations, but it's not friendly to users for how to configure it in the `broker.conf`. Could you use an `enum` instead of `null` for it?

```java
    private ServiceUnitTableViewSyncerType loadBalancerServiceUnitTableViewSyncer = ServiceUnitTableViewSyncerType.None;

    public enum ServiceUnitTableViewSyncerType {
        None,
        MetadataStoreToSystemTopicSyncer,
        SystemTopicToMetadataStoreSyncer
    }
```

Then we can update the `broker.conf` again to include this config.

Reply:
Besides, you should update the PIP document in this PR to keep implementation and documentation consistent. I think it's acceptable because it's not a big change.

Reply:
sure. let me update this.